sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,219 @@
+ - Resolved a pinned host memory specific performance regression on CUDA that
+   was introduced with the host unified behavior dependent logic [3be63ab]
+ - Fixed illegal accesses that could happen when an application that uses host
+   tasks exits without waiting for host tasks completion [552a521]
+ - Fixed `sycl::event::get_info` queries that were working incorrectly when
+   called on event without an encapsulated native handle [5d5a792]
+ - Fixed compilation error with using multidimensional subscript for
+   `sycl::accessor` with atomic access mode [0bfd34e]
+ -  Fixed a crash that happened when an accessor passed to a reduction was
+    destroyed immediately after [b80f13e]
+ - Fixed `sycl::device::get_info` with `sycl::info::device::max_mem_alloc_size`
+   which was returning incorrect value in case of Level Zero backend [8dbaa53]
+
+## API/ABI breakages
+- None
+
+## Known issues
+ - GlobalWorkOffset is not supported by Level Zero backend [6f9e9a76]
+ - User-defined functions with the same name and signature (exact match of
+   arguments, return type doesn't matter) as of an OpenCL C built-in
+   function, can lead to Undefined Behavior.
+ - A DPC++ system that has FPGAs installed does not support multi-process
+   execution. Creating a context opens the device associated with the context
+   and places a lock on it for that process. No other process may use that
+   device. Some queries about the device through device.get_info<>() also
+   open up the device and lock it to that process since the runtime needs
+   to query the actual device to obtain that information.
+ - On Windows, DPC++ compiler enforces using dynamic C++ runtime for
+   application linked with SYCL library by:
+    - linking with msvcrt[d].dll when `-fsycl` switch is used;
+    - emitting an error on attempts to compile a program with static C++ RT
+      using `-fsycl` and `/MT` or `/MTd`.
+   That protects you from complicated runtime errors caused by C++ objects
+   crossing sycl[d].dll boundary and not always handled properly by different
+   versions of C++ RT used on app and sycl[d].dll sides.
+ - The format of the object files produced by the compiler can change between
+   versions. The workaround is to rebuild the application.
+ - Using `sycl::program`/`sycl::kernel_bundle` API to refer to a kernel defined
+   in another translation unit leads to undefined behavior
+ - Linkage errors with the following message:
+   `error LNK2005: "bool const std::_Is_integral<bool>" (??$_Is_integral@_N@std@@3_NB) already defined`
+   can happen when a SYCL application is built using MS Visual Studio 2019
+   version below 16.3.0 and user specifies `-std=c++14` or `/std:c++14`.
+ - Printing internal defines isn't supported on Windows [50628db]
+ - Group algorithms for MUL/AND/OR/XOR cannot be enabled for group scope due to
+   SPIR-V limitations, and are not enabled for sub-group scope yet as the
+   SPIR-V version isn't automatically raised from 1.1 to 1.3 [96da39e]
```

Comment:
Haven't we raised spir-v at #3267

Reply:
@v-klochkov Could you please confirm that this "known issue" can be removed?

Reply:
Yes, it can be removed. Thank you.